scripts/run-executable.ps1

```diff
@@ -86,6 +92,19 @@ if (!(Test-Path $Path)) {
     Write-Error "$($Path) does not exist!"
 }
 
+# Validate the code coverage switch
+if ($CodeCoverage) {
+    if (!$IsWindows) {
+        Write-Error "-CodeCoverage switch only supported on Windows";
```

Comment:
FWIW gcc/clang support their own coverage modes, `--coverage`, which can output coverage reports as well on Linux.

Reply:
We want to make sure we use a format that Azure Pipelines supports, which is what currently drives this PR.

Reply:
At some point in the future, if gcc/clang can output coverage in cobertura format, it would be nice to also collect linux coverage. However, at this time, AZP only supports reporting 1 coverage file per pipeline, so we'd have to merge the Windows and linux coverage, which is undesirable.

Reply:
That's rather unfortunate. It would be nice to see how much coverage spinquic is actually getting, or how much the unit tests on their own are getting.